.gitignore

```diff
@@ -1,4 +1,4 @@
 vendor/
 composer.lock
 tests/FacebookTestCredentials.php
-
```

Comment:
You should remove this, use [global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for IDE ignored files